README.md

```diff
@@ -1 +1,156 @@
-# recipeApp
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
For recipe , i you have suggested , i create the logo and add it to the ReadMe